go/vt/vtctl/reparentutil/reparent_sorter.go

```diff
@@ -71,6 +74,14 @@ func (rs *reparentSorter) Less(i, j int) bool {
 		return true
 	}
 
+	// We want tablets backing up always at the end of the list, so that's the first thing we check
+	if !rs.backingUp[topoproto.TabletAliasString(rs.tablets[i].Alias)] && rs.backingUp[topoproto.TabletAliasString(rs.tablets[j].Alias)] {
+		return true
+	}
+	if rs.backingUp[topoproto.TabletAliasString(rs.tablets[i].Alias)] && !rs.backingUp[topoproto.TabletAliasString(rs.tablets[j].Alias)] {
+		return false
+	}
+
```

Comment:
This check should be after the position and promotion rule. If a tablet is more advanced it should always come first in the list.

Reply:
Fixed.